charts/datree-admission-webhook/values.yaml

```diff
@@ -1,26 +1,29 @@
 # Default values for datree-admission-webhook.
 
-# The name of the namespace all resources will be created in.
-namespace: datree
+# The name of the namespace all resources will be created in, if not specified in the release.
+namespace: ""
 
 # The number of Datree webhook-server replicas to deploy for the webhook.
 replicaCount: 2
 
 # Additional labels to add to all resources.
 customLabels: {}
 
+# Additional annotations to add to all resources.
+customAnnotations: {}
```

Comment:
😬
I think that we need to allow to change annotations to every resource. But we can do it in the future. It's not urgent in my opinion